lib/gltest/window.ex

```diff
@@ -270,36 +216,74 @@ defmodule GlTest.Window do
     state
   end
 
-  defp draw(%{frame: frame} = state) do
-    :gl.clearColor(0.2, 0.3, 0.3, 1.0)
-    :gl.clear(:gl_const.gl_color_buffer_bit())
+  defp draw(%{frame: frame, shader_program: shader_program} = state) do
+    :gl.enable(:gl_const.gl_depth_test())
+    :gl.depthFunc(:gl_const.gl_less())
 
-    scale_transform(state.shader_program)
+    :gl.clearColor(0.2, 0.3, 0.3, 1.0)
+    :gl.clear(:gl_const.gl_color_buffer_bit() ||| :gl_const.gl_depth_buffer_bit())
 
     :gl.activeTexture(:gl_const.gl_texture0())
     :gl.bindTexture(:gl_const.gl_texture_2d(), state.texture1)
     :gl.activeTexture(:gl_const.gl_texture1())
     :gl.bindTexture(:gl_const.gl_texture_2d(), state.texture2)
-    :gl.bindVertexArray(state.rect_vao)
-    :gl.drawElements(:gl_const.gl_triangles(), 6, :gl_const.gl_unsigned_int(), 0)
+
+    rads = :math.sin(Gltest.Math.radians((now() - state.start_time) / 1_000)) * 90
+
+    model =
+      Graphmath.Mat44.multiply(
+        Graphmath.Mat44.multiply(
+          Graphmath.Mat44.identity(),
+          Graphmath.Mat44.make_rotate_x(rads)
+        ),
```

Comment:
```suggestion
  Graphmath.Mat44.make_rotate_x(rads)
```

No reason to multiply by the identity matrix I believe.